### PR TITLE
PIMS-1318

### PR DESCRIPTION
**Pull request by devinleighsmith**
Add modal popups when loading/saving parcel detail form drafts.

### CODE REVIEW

File: frontend/src/OnLoadActions.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  const history = useHistory();
+
+  return (
+    <GenericModal
+      title="Unsaved Draft"
+      message="You have an unsaved Property Draft, would you like to resume editing it?"
+      cancelButtonText="Discard"
+      okButtonText="Resume Editing"
+      display={
+        history.location.pathname !== '/submitProperty' && isStorageInUse(PARCEL_STORAGE_NAME)
```
Comment (asanchezr) [inline frontend/src/OnLoadActions.tsx:16]: quick check - weren't we going to rename `/submitProperty` to `/property` ? or that's in another PR?

Comment (Fosol) [reply frontend/src/OnLoadActions.tsx:16]: I asked Jonny to do this originally.

**Comment by asanchezr** [comment]
This branch is showing as out-of-date in GH. Could you please rebase? Thanks

File: frontend/src/components/common/FormikPersist.tsx [hunk 2]
```diff
@@ -11,59 +13,119 @@ export interface PersistProps {
...
   writeOnly?: boolean;
+  loadDraft?: boolean;
+}
+interface PersistState {
+  showLoadDraftDialog: boolean;
+}
+
+// Do not allow stored data to be loaded if the application version does not match.
+interface VersionedStorage {
+  version: string;
```
Comment (Fosol) [inline frontend/src/components/common/FormikPersist.tsx:24]: How do we set this value?

File: frontend/src/OnLoadActions.tsx (hunk 1 above)
Comment (asanchezr) [reply frontend/src/OnLoadActions.tsx:16]: okay

File: frontend/src/components/common/FormikPersist.tsx (hunk 2 above)
Comment (devinleighsmith) [reply frontend/src/components/common/FormikPersist.tsx:24]: Currently I'm using the value set in package.json
All this means is that whenever we bump the version in a frontend release people will lose their drafts.

Comment (asanchezr) [reply frontend/src/components/common/FormikPersist.tsx:24]: I like this. Would love to see a comment somewhere for reference when we need it.

Comment (devinleighsmith) [reply frontend/src/components/common/FormikPersist.tsx:24]: Fair, will add in my next PR with the building updates.

**Pull request merged**